website/source/docs/builders/azure.html.md

```diff
@@ -83,6 +83,19 @@ builder.
     configures your Tenant ID, Object ID, Key Vault Name, Key Vault Secret, and
     WinRM certificate URL.
 
+-   `virtual_network_name` (string) Use a pre-existing virtual network for the VM.  This option enable private
+    communication with the VM, no public IP address is **used** or **provisioned**.  This value should only be set if
+    Packer is executed from a host on the same subnet as the virtual network.
```

Comment:
-> .. on the same subnet / virtual network.
 or
..  on the same subnet in the same virtual network.